sdv/metadata/multi_table.py

```diff
@@ -427,6 +432,92 @@ def add_constraint(self, table_name, constraint_name, **kwargs):
         table = self._tables.get(table_name)
         table.add_constraint(constraint_name, **kwargs)
 
+    def _validate_single_table(self, errors):
+        for table_name, table in self._tables.items():
+            try:
+                table.validate()
+            except Exception as error:
+                errors.append('\n')
+                tittle = f'Table: {table_name}'
+                error = str(error).replace(
+                    'The following errors were found in the metadata:\n', tittle)
+                errors.append(error)
+
+    def _validate_disjoined_tables(self, parent_map, child_map):
```

Comment:
can we rename to match the positive case? so `_validate_all_tables_connected` or something like that?